packages/agreement-process/src/services/agreementService.ts

```diff
@@ -105,13 +103,25 @@ export function agreementServiceBuilder(
       );
       return await repository.createEvent(createAgreementEvent);
     },
-    async getConsumers(
```

Comment:
@MalpenZibo for consistency, I renamed this function that you created in #125 into `getAgreementConsumers`, also to maintain the same name from the Scala codebase and the convention we are using in this file. You could port it to your PR, and then I rebase